docs/tutorials/multimodal/object_detection/prepare_coco17.md

```diff
@@ -0,0 +1,133 @@
+# AutoMM Detection - Prepare COCO2017 Dataset
+:label:`sec_automm_detection_prepare_coco17`
+
+[COCO](https://cocodataset.org/#home) is a large-scale object detection, segmentation, and captioning dataset. 
+For detection, the most commonly used version is COCO17 with the largest number of training data.
+There are 80 classes, 123,287 images, 886,284 instances, and the median image ratio is 640 x 480.
+This tutorial will walk through the steps of preparing this dataset for Autogluon MultiModalPredictor.
+
+You need 42.7 GB disk space to download and extract this dataset. SSD is preferred over HDD because of its better performance.
```

Comment:
will it be slow to run given it's large size?

Reply:
Do you mean it's slow building the tutorial or it's slow for user to download?